.env

```diff
@@ -0,0 +1 @@
+'DATABASE_URL="postgresql://postgres:password@localhost:5432/nerpm?schema=public"'
```

Comment:
This file should be deleted. Your env file should be in your backend!

Reply:
Hi where exactly can I find this 'DATABASE_URL="postgresql://postgres:password@localhost:5432/nerpm?schema=public"' line? I can't find the folder that it's in